schedule/storage/nfs.yaml

```diff
@@ -15,8 +15,10 @@ conditional_schedule:
         - kernel/nfs_server
       nfs_client:
         - kernel/nfs_client
+    VERSION:
+      15-SP6:
+        - kernel/nfs_stress_ng
 schedule:
   - boot/boot_to_desktop
   - kernel/before_nfs_test
   - '{{nfstest}}'
-  - kernel/nfs_stress_ng
```

Comment:
Please make separate condition for nfs_stress_ng to don't mix it with server/client logic.